src/components/MessagesList/MessagesGroup/Message/MessagePart/FilePreview.vue

```diff
@@ -20,14 +22,15 @@
 -->
 
 <template>
-	<a :href="link"
-		class="container"
-		:class="{ 'is-viewer-available': isViewerAvailable }"
-		target="_blank"
-		rel="noopener noreferrer"
-		@click="showPreview">
-		<img v-if="!isLoading && !failed"
+	<file-preview v-bind="filePreview"
```

Comment:
This file really bothers me. The shared part is actually so little, that it should just be 2 maintainable subcomponents instead.